api/controllers/webhook.js

```diff
@@ -44,6 +30,30 @@ const findSiteForWebhookRequest = (request) => {
     });
 };
 
+const organizationWebhookRequest = async (payload) => {
+  const {
+    action, membership, organization,
+  } = payload;
+  const { login: orgName } = organization;
+
+  const { login } = membership.user;
+  const username = login.toLowerCase();
+  const user = await User.findOne({ where: { username } });
+
+  if (orgName === config.federalistUsers.orgName) {
+    if (['member_added', 'member_removed'].includes(action)) {
+      EventCreator.audit(Event.labels.FEDERALIST_USERS_MEMBERSHIP, user || User.build({ username }), payload);
```

Comment:
1. I still don't understand the advantage of creating the event without the user, when we can do it with the user. Github already has an audit trail of these events, what our audit events are tracking are changes to the things INSIDE of our system and the expectation (at least my expectation) is that audit events will always tie back to a model instance, I actually thought we had put in a validation to require this, but now I don't see it.
For example, if we leave this as is and then query "everything that happened to user X", this event will not show up.

2. Also consider returning early here for readability and doing a bit less work, ex:
```
  if (orgName !== config.federalistUsers.orgName
      || !['member_added', 'member_removed'].includes(action)) {
    return;
  }

  const { login } = membership.user;
  const username = login.toLowerCase();
  const user = await User.findOne({ where: { username } });
  ...
```

Reply:
@davemcorwin 
1. i have switched to creating the event after the user exists ... no longer with empty user
2. done